libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxAudioGuidance.kt

```diff
@@ -206,11 +212,17 @@ internal constructor(
 
         /**
          * Construct an instance without registering to [MapboxNavigationApp].
+         *
+         * @param options Optional [MapboxSpeechApiOptions]
          */
         @JvmStatic
-        fun create() = MapboxAudioGuidance(
+        @JvmOverloads
+        fun create(
```

Comment:
This is invoked from `getRegisteredInstance`, which doesn't accept an `options` parameter.
If you follow the chain further, you'll see that it's impossible to set such an option when using Drop-In UI.
We should add it to `NavigationViewOptions` and pass all the way down to `MapboxSpeechAPI`.
Again, it can be done in a follow-up PR and release: it won't break anything, it'll just ad more customization possibilities.

Reply:
Thanks, good note!

> This is invoked from getRegisteredInstance

If we add options as an argument (`fun getRegisteredInstance(options: MapboxSpeechApiOptions)`) then options might be ignored if another instance of `MapboxAudioGuidance` was already created without options. I don't like that much, but not sure if we have choice. Let's discuss this.

Reply:
OK, then how about we don't pass it as a constructor parameter to `MapboxAudioGuidance`, but rather only to `MapboxAudioGuidanceVoice`? We already recreate `MapboxAudioGuidanceVoice` when language changes, so it makes sense to also recreate it when gender changes. While `MapboxAudioGuidance` doesn't really have to have this as a property. It only passes it further to `MapboxAudioGuidanceVoice`.

Reply:
I've added a new function `updateSpeechApiOptions()` which can be used to update current options, including options in `getRegisteredInstance()`. This looks like the best option for now, given that `getRegisteredInstance` can be called from different places in different modules.